reg_tests/CTestList.cmake

```diff
@@ -238,7 +238,7 @@ else(NOT ENABLE_CUDA)
   #=============================================================================
   add_test_r(ablNeutralNGPTrilinos 2)
   add_test_r(conduction_p4 2)
-  add_test_r(airfoilRANSEdgeTrilinos 1)
+  #add_test_r(airfoilRANSEdgeTrilinos 1)
```

Comment:
This removes capability introduced in dbe7646292e0d4bb54950bf12799ed4f3ed801d8 Please undo this change

Reply:
fixed